query/stat_processor.go

```diff
@@ -8,16 +8,40 @@ import (
 )
 
 // statProcessor is used to collect, analyze, and print query execution statistics.
-type statProcessor struct {
-	prewarmQueries bool       // PrewarmQueries tells the StatProcessor whether we're running each query twice to prewarm the cache
-	c              chan *Stat // c is the channel for Stats to be sent for processing
-	limit          *uint64    // limit is the number of statistics to analyze before stopping
-	burnIn         uint64     // burnIn is the number of statistics to ignore before analyzing
-	printInterval  uint64     // printInterval is how often print intermediate stats (number of queries)
-	wg             sync.WaitGroup
+type statProcessor interface {
+	getArgs() *statProcessorArgs
+	sendStats(stats []*Stat)
+	sendStatsWarm(stats []*Stat)
```

Comment:
Since we are refactoring, maybe we should consider removing some of the stutter in the interface there. The word `Stats` seems superfluous in the function name.

`b.sp.sendStats(stats)` => `b.sp.send(stats)`

Same goes for `sendStats` and `sendStatsWarm`.

Reply:
I like this suggestion.
`sendStats` -> `send`
`sendStatsWarm` -> `sendWarm`